### PR TITLE
Update license to AGPL

### PR DESCRIPTION
**Pull request by Neo-Desktop**
As per contributor quorum on 17/06/2023 - update license to AGPL

This PR also adds known history to files

### CODE REVIEW

**Review by CONIGUERO — APPROVED** [review]
Would rebrand to UMSKT, aside from that I approve

**Review by thepwrtank18 — CHANGES_REQUESTED** [review]
We should change WindowsXPKg to UMSKT, since that's the new name now (and do the same for everything else in the GitHub repo).

File: src/BINK1998.cpp [hunk 1]
```diff
@@ -1,6 +1,31 @@
-//
-// Created by Andrew on 01/06/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/BINK1998.cpp:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/BINK2002.h [hunk 2]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by neo on 6/6/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/BINK2002.h:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/cli.cpp [hunk 3]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by Andrew on 01/06/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/cli.cpp:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/cli.h [hunk 4]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by neo on 6/6/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/cli.h:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/confid.cpp [hunk 5]
```diff
@@ -1,6 +1,31 @@
-//
-// Created by WitherOrNot on 06/02/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/confid.cpp:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/confid.h [hunk 6]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by neo on 6/6/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/confid.h:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/header.h [hunk 7]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by neo on 5/26/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/header.h:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/key.cpp [hunk 8]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by neo on 5/26/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/key.cpp:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/main.cpp [hunk 9]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by Andrew on 01/06/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/main.cpp:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

File: src/util.cpp [hunk 10]
```diff
@@ -1,6 +1,24 @@
-//
-// Created by Andrew on 01/06/2023.
-//
+/**
+ * This file is a part of the WindowsXPKg Project
+ *
+ * Copyleft (C) 2019-2023 WindowsXPKg Contributors (et.al.)
```
Comment (thepwrtank18) [inline src/util.cpp:4]:
```suggestion
 * Copyleft (C) 2019-2023 UMSKT Contributors (et.al.)
```

**Review by WitherOrNot — COMMENTED** [review]
Echoing @thepwrtank18, everything looks good except the project name

File: README.md [hunk 11]
```diff
@@ -1,13 +1,13 @@
...
-[![Zulip chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://umskt.zulipchat.com) 
+[![Zulip chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://umskt.zulipchat.com)
 [![libera.chat - #mspid](https://img.shields.io/badge/libera.chat-%23mspid-brightgreen)](https://web.libera.chat/gamja/?nick=Guest?#mspid)
 
 ### **Plan of Action / ToDo List**
 
 In light of the recent exponential interest in this project I've decided to put updates of this project here:
 
-* [Please see ticket #8 for more information](https://github.com/Neo-Desktop/WindowsXPKg/issues/8)
+* [Please see ticket #8 for more information](../../issues/3)
```
Comment (WitherOrNot) [inline README.md:10]: incorrect issue number

**Review by thepwrtank18 — APPROVED** [review]
All clear from what I see.

**Review by WitherOrNot — CHANGES_REQUESTED** [review]

File: README.md (hunk 11 above)
Comment (WitherOrNot) [reply README.md:10]: should be #8

**Review by WitherOrNot — APPROVED** [review]
All links, badges work, gud2go

**Review by CONIGUERO — APPROVED** [review]

**Comment by thepwrtank18** [comment]
image
verification that @Endermanch approved it

**Review by Endermanch — COMMENTED** [review]
👍

**Pull request merged**